findspam.py

```diff
@@ -99,7 +101,7 @@ def malicious_link(s, site, *args):
     try:
         parsed_href = tld.get_tld(href, as_object=True)
         print(parsed_href.domain, SE_SITES_DOMAINS)
-        if parsed_href.tld in SE_SITES_DOMAINS:
+        if site == 'stackoverflow.com' and parsed_href.tld in SE_SITES_DOMAINS:
```

Comment:
I think you put this in the wrong place... Should it be with SAFE_EXTENSIONS on 117?

Reply:
My bad...